packages/create-redwood-app/template/package.json

```
@@ -15,7 +15,7 @@
     "root": true
   },
   "engines": {
-    "node": ">=14 <16",
+    "node": "14.x",
     "yarn": ">=1.15"
```

Could we lock this down to `1.x` too?
```suggestion
    "yarn": "1.x"
```

---

@dac09 Did you mention that there's a particular version of yarn that doesn't have workspace issues?

---

Yep! 1.18 works for the resolution issues

---

Is there an issue or something that I can read regarding that?

---

I think we should pin yarn to that version, afaik it's 100% transparent to users

---

Yup -> https://github.com/yarnpkg/yarn/issues/7807

And here's my suggested work around (in reference to rwt link) 

During our discussion last week, the suggestion was to not pin it to 1.18 but to run the set policies command for them (with a confirmation prompt), when they run `rw upgrade`

---

I believe its the same problem, that surfaces in 2 different situations: 
a) when running `rw upgrade`
b) and also during the contributor workflow with `rwt link`

---

> During our discussion last week, the suggestion was to not pin it to 1.18 but to run the set policies command for them (with a confirmation prompt), when they run rw upgrade

🤔 I don't understand why we would be doing that? If this solves an issue then we should just do it - the change is transparent to the user, and it's only scoped to the local project.

Do you know what the motivation was?

---

I kinda forgot 🙈. The meet was recorded though, so once California is up we can review the recording? I'm also for setting the version here!

---

IIRC it was because the error message if you have the wrong version of yarn wasn't helpful enough. It would just refuse to start, right? Where as if we run the policies command it'll download the correct version of yarn for the user and everything will work much smoother